src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -204,6 +204,18 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
      */
     @Override
     public void validateArguments() {
+        // throw if any filter is specified twice
+        final List<String> moreThanOnce = userReadFilterNames.stream()
+                .collect(Collectors.groupingBy(e -> e, Collectors.counting()))
+                .entrySet().stream().filter(e -> e.getValue() != 1)
+                .map(e -> e.getKey() + " (" + e.getValue() + ")")
+                .collect(Collectors.toList());
```

Comment:
I did that for warn the user how many times they specified the filter. With a long command line, it could happen that it was more than two and the user will need to check the command line too much. I will change it anyway, but it will be nicer to have the counts.